pinc/forum_interface.inc

```diff
@@ -58,11 +58,12 @@ function phpbb_lang($locale=FALSE)
         {
             $query = sprintf("
                 SELECT *
-                FROM ${phpbb_table_prefix}_lang
-                WHERE lang_iso='%s'
-                ", mysqli_real_escape_string(DPDatabase::get_connection(), $lang));
-            $res = mysqli_query(DPDatabase::get_connection(), $query) or
-                die(DPDatabase::log_error());
+                FROM %s_lang
```

Comment:
Keep this directly in the string since it's a table name.

Reply:
Ditto everywhere else.

Reply:
Hopefully phpbb_table(...) helps a lot.

Reply:
This is fine, but I'm not sold that it's better than what was there before. Our best practices keeps the table names inside the string itself so the SQL is more readable to humans who care a lot more about the column and table names (the structure of the query) rather than the values: https://www.pgdp.net/wiki/DP_Code_Best_Practices#Validating_project_tables